src/main/java/log_out/ui/LogOutView.java

```diff
@@ -0,0 +1,48 @@
+package log_out.ui;
+
+import log_out.interface_adapters.LogOutController;
+import log_out.interface_adapters.LogOutControllerData;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyChangeListener;
+
+public class LogOutView extends JPanel implements PropertyChangeListener, ActionListener {
+
+    private final LogOutController controller;
+
+    public JButton logOutButton = new JButton("logOutButton");
+
+    public LogOutView(LogOutController controller){
+        this.controller = controller;
+
+        logOutButton.addActionListener(this);
+    }
+
+    @Override
+    public void propertyChange(PropertyChangeEvent evt) {
+        if (evt.getPropertyName().equals("Log Out")){
+            String logOut = "log out";
```

Comment:
not sure what this code is supposed to lead to, but from what I imagine a successful log out would look like, the main view would close and the log in page should be brought up again maybe?